contracts/script/RegisterOperator.s.sol

```diff
@@ -65,7 +65,22 @@ contract RegisterOperator is Script {
         BN254.G2Point pk2;
     }
 
+    function readIPConfig(string memory operatorId) internal view returns (string memory) {
+        string memory root = vm.projectRoot();
+        string memory path = string.concat(root, "/docker/eigenlayer/config.json");
```

Comment:
Not sure this is the best pattern, may cause issues later, but for reference best practice is to add the path as an arg as well 
espc if we ever migrate this to anything other than docker , this will break

Reply:
can ignore for now